features/bootstrap/DatasetViewContext.php

```diff
@@ -413,10 +413,7 @@ public function iShouldNotSeeTabWithTable($arg1, TableNode $table)
      */
     public function iHaveAddedLinkToDataset($arg1, $arg2, $arg3)
     {
-        if ("3D Viewer" == $arg1 ) {
```

Comment:
@kencho51 since I deleted the 3D Models test, I am also deleting this switch case